README.md

```diff
@@ -45,6 +45,40 @@ python3 -m http.server 8000 --directory out/Default/gen/front_end
 
 The frontend will be available at `http://localhost:8000/inspector.html` (or `http://localhost:8000/rn_inspector.html` for the RN-specific entry point).
 
+### Syncing with the upstream repo
+
+Periodically, we will sync this project with the upstream [ChromeDevTools/devtools-frontend](https://github.com/ChromeDevTools/devtools-frontend) repo. We always update our fork from a stable upstream branch.
+
+#### Viewing the last synced version
+
+```sh
+git tag --merged <branch>
+```
+
+#### Performing a repo sync
+
+1. (One-time prerequisite) Add a Git remote pointing to [ChromeDevTools/devtools-frontend](https://github.com/ChromeDevTools/devtools-frontend).
+
+    ```sh
+    git remote add google git@github.com:ChromeDevTools/devtools-frontend.git
+    ```
+
+    > Note: The `google` naming is optional — `upstream` will typically be set already when originally cloning your local repo with `gh repo clone`.
+
+2. Merge `main` with the target upstream branch, then resolve all conflicts(!).
+
+    ```sh
+    git switch main
+    git fetch --all
+    git merge google/chromium/5845
+    ```
+
+3. Tag the merge commit! This marks the point at which we synced the repo with the upstream branch.
+
+    ```sh
+    git tag chromium/5845
```

Comment:
This feels ambiguous. Shouldn't `chromium/5845` mean the same thing across repos, i.e. the upstream revision we're "based" on? And then the merge commit would be a *child* of that commit, warranting its own distinct tag.

Reply:
I've pushed and suggested `sync-chromium-5845` as the format.